### ISSUE TITLE
Add ability to filter by "events" in issue widgets 

### ISSUE THREAD

**Issue by gotenxds** [getsentry/sentry#68249]
### Problem Statement

Today when creating an issue widget you can add the number of events as a column - this column is a `count` column.
But for some reason you cannot filter by it, that is you cannot filter events>x for example

![image](https://github.com/getsentry/sentry/assets/3519520/57a6ed74-e845-4e97-9506-ea6dcd9c61a4)

### Product Area

Dashboards

**Comment by gggritso** [comment]
@gotenxds thanks for raising this! The "filter" field is to filter out the _events_ associated with the issues. The widget you're showing is a count of events per issue ID. You can use the filters to, for example, filter out events for a certain user or a transaction, which would affect the event counts.

There's no way to filter out the _final results_ of the table. Is that what you're looking for? I imagine it's something like "only show issues with > x events"

**Comment by gotenxds** [comment]
Yes, my goal here was to try and show only issues that had x events over the period of time selected

**Comment by gggritso** [comment]
Understood, thanks. This feature makes sense! I'm going to add this to our backlog. Not sure if we'll be able to work on it soon! Keeping the issue open, for tracking
